import_activity_events.py

```diff
@@ -119,19 +125,29 @@
       ua_browser,
       ua_version,
       ua_os
-    FROM temporary_raw_activity_data;
+    FROM (
+      SELECT *, ROW_NUMBER() OVER(ORDER BY timestamp, uid, type) AS row
+      FROM temporary_raw_activity_data
+    )
+    WHERE row % {modulo} = 0;
```

Comment:
I think doing this by row number will make it hard to analyse the data, because you'll have some-but-not-all of the events from any particular user.  That makes it hard to do e.g. meaningful retention analysis, since we wouldn't know whether the user stopped using firefox, or just happend to be outside the sample set.

It may be better to sample 10% of userids rather than 10% of events, like we used to do in the S3 export.  We could use something like this to convert the uid to an integer for sampling"

```
 SELECT uid, STRTOL(SUBSTRING(uid FROM 0 FOR 8), 16) % 100 AS sampleNum FROM activity_events LIMIT 10;
                               uid                                | samplenum 
------------------------------------------------------------------+-----------
 b0ae7947a2aff25fdd19f55eb948a1eecda728c4c0e6eb3538c715deb5e5be8d |        20
 7653f1590e47dbbeaa9110c979b7159c361b69c4cef04a8cfa029a3ede93d9c6 |        97
 a4f7f93ea4c9edace3b630c51c3e999fb11fa041036035c030589fd002ac0034 |        63
 c8ab8bf2685a29c72d3a8d6f1eba7388573cb813b1701cac688f5bae24ace47f |        55
 58960ea4307b7738694a554ea76110f04e1735160b858a29c9ab64e9e0cb6364 |        22
 5d904e947dee5c84226cb7d76695c0edb688b13fdd94404200b3f1aa219da784 |        49
 d487c8ff86788a3a02eeb0f2d859f1f914ba40dc6534d9c3bc484760f16942e6 |        87
 705f2f5098db22514cdf097845675755f0f9e035f7fdf0da497be9c889517193 |        89
 17947fc6718702699e273d042eeec46c779be4087060a123eea303a996d664ed |         0
 6eb1f20b4f400b13e7196c1ea2e6a8c06962113c8a58c9df3f3cb4a4951031c9 |        24
(10 rows)
```